nfsn-cron.sh

```diff
@@ -17,8 +17,8 @@ if find certs -name cert.pem -type l \
 then
 	echo " + Certificte will expire in 30 days or less! SSH into this site and"
 	echo "   run the following commands to renew your certificates:"
-	echo "	cd ~/lets-nfsn.sh/letsencrypt.sh/"
-	echo "	./letsencrypt.sh --cron"
+	printf '	cd %q\n' "$PWD"
+	echo   '	./letsencrypt.sh --cron'
```

Comment:
Actually, if we're doing that we can replace the whole thing with the following, and cut out the unnecessary `cd`:

```sh
	printf '\t%q --cron\n' "$(realpath letsencrypt.sh)"
```